content/profiles/Matthias_Beyer.md

```diff
@@ -0,0 +1,40 @@
+---
+name: "Matthias Beyer"
+kind: :member
+enteryear: 2013
+tags:
+  - archlinux
+  - bash
+  - c
+  - c++
+  - debian
+  - git
+  - github
+  - gpg
+  - haskell
+  - i3
+  - kernel
+  - latex
+  - linux
+  - mpd
+  - mutt
+  - ncmpcpp
+  - nix
+  - nixos
+  - opensource
+  - privacy
+  - racket
+  - rails
+  - ruby
+  - tiling-windowmanager
+  - tmux
+  - vim
+---
+
+I entered the university in 2013, summer semester. I've been a member of the
+unfug from the very start of my first semester and I took the leadership of
+the unfug in my 4th semester. It was always a pleasure to me to hear talks
+with the people at the weekly meetups and although I had several topics for
```

Comment:
Good point. We shouldn't restrict members, anyways. I would say, english is fine if a member can provide her/his page in english, but if she/he only wants to write it in german, we go for it. Ack?